aviatrix/resource_aviatrix_gateway.go

```diff
@@ -385,6 +386,20 @@ func resourceAviatrixGateway() *schema.Resource {
 				DiffSuppressFunc: DiffSuppressFuncString,
 				Description:      "A list of monitored instance ids separated by comma when 'monitor gateway subnets' feature is enabled.",
 			},
+			"idle_timeout_value": {
```

Comment:
I think `idle_timeout` would be a better name. Adding `_value` to the end is unnecessary.